docs/npm.md

```diff
@@ -44,27 +46,75 @@ sudo yum install -y python2
 npm config set python python2
 ```
 
+Proceed to [installing](#installing)
+
 ## Alpine
 
 ```bash
 apk add alpine-sdk bash libstdc++ libc6-compat
 npm config set python python3
 ```
 
+Proceed to [installing](#installing)
+
 ## macOS
 
 ```bash
 xcode-select --install
 ```
 
+Proceed to [installing](#installing)
+
 ## FreeBSD
 
 ```sh
 pkg install -y git python npm-node14 yarn-node14 pkgconf
 pkg install -y libinotify
 ```
 
-## Issues with Node.js after version upgrades
+Proceed to [installing](#installing)
+
+## Windows
+
+Installing code-server requires all of the [prerequisites for VS Code development](https://github.com/Microsoft/vscode/wiki/How-to-Contribute#prerequisites). When installing the C++ compiler tool chain, we recommend using "Option 2: Visual Studio 2019" for best results.
```

Comment:
I ran into several issues installing `windows-build-tools` on a fresh Windows machine.

Reply:
This sounds kinda rough. I have some experience building stuff on Windows (actually doing Clang/LLVM + MSVC, which I think can be cross-compiled under Wine) so this might be something I can look into at some point...

Reply:
Yeah, installing Visual Studio is an unfortunate prerequisite, where `npm install -g windows-build-tools` would be a bit more understandable.

If you think it'd be helpful, I can do some more debugging with the npm method and see if there are any pointers I can leave in this doc, or put upstream.